lib/providerAPI/bridgeProviderAPI/implement/implementOpalAPI.ts

```diff
@@ -1,42 +1,22 @@
-import { ImplementorAPI } from './implementorAPI';
+import ImplementorAPI from './implementorAPI';
 
 export class ImplementOpalAPI extends ImplementorAPI {
   async impGetCollection(collectionId) {
-    const collection = await this._api.rpc.unique.collectionById(collectionId);
-    return this.toObject(collection);
+    const result = await this.api.rpc.unique.collectionById(collectionId);
```

Comment:
Why do you rename collection to result? 

Because result is not collection and collection is result.value?

Reply:
Yeap. The type of the result here is `Option<UpDataStructsRpcCollection>`. I think it is not correct to call it `collection`.